src/test/ui/span/issue-39018.stderr

```diff
@@ -1,26 +1,36 @@
 error[E0369]: binary operation `+` cannot be applied to type `&str`
-  --> $DIR/issue-39018.rs:2:13
+  --> $DIR/issue-39018.rs:2:22
    |
 LL |     let x = "Hello " + "World!";
-   |             ^^^^^^^^^^^^^^^^^^^ `+` can't be used to concatenate two `&str` strings
+   |             ---------^---------
+   |             |          |
+   |             |          &str
+   |             &str
+   |             `+` can't be used to concatenate two `&str` strings
```

Comment:
Can you change the span for the explanation to use the span pointing at the operator?

Reply:
I think this is fixed? Not sure if issue-47380.stderr looks right though